config/config-mp/src/test/java/io/helidon/config/mp/MpConfigTest.java

```diff
@@ -274,5 +275,29 @@ public static void reset() {
             counter.set(0);
         }
     }
+
+    @Test
+    void testEnvVar() {
+        ConfigProviderResolver instance = ConfigProviderResolver.instance();
+        ClassLoader myCl = Thread.currentThread().getContextClassLoader();
+        Config current = ConfigProvider.getConfig(myCl);
+
+        try {
+            instance.registerConfig(instance.getBuilder()
+                                            .withSources(MpConfigSources.environmentVariables())
+                                            .build(),
+                                    myCl);
+            Config myConfig = instance.getConfig(myCl);
+            // this must not throw an exception - path should be on any environment
+            // and the MP env var processing should make it available
+            String path = myConfig.getValue("path", String.class);
+
+            io.helidon.config.Config helidonConfig = (io.helidon.config.Config) myConfig;
+            // should work if we use it as SE as well
+            helidonConfig.get("path").asString().get();
```

Comment:
Can we assert those resolved PATH values equals with `System.getenv("PATH")`?

Reply:
Using custom env var, validating content.